src/Promenade.Controllers/NewsController.cs

```diff
@@ -24,13 +24,20 @@ protected override PageType PageType
         {
         }
 
-        [Route("{stub?}")]
+        [HttpGet("{stub?}")]
         public async Task<IActionResult> Page(string stub)
         {
             return await ReturnPageAsync(stub);
         }
 
-        [Route("{stub?}/item/{id}")]
+        [HttpPost("{stub?}")]
+        public async Task<IActionResult> PagePreview(string stub)
+        {
+            return await ReturnPreviewPageAsync(stub,
+                HttpContext.Request.Query["PreviewId"]);
+        }
```

Comment:
This should be added to the HomeController as well, I think layout pages should be able to work with the home page.

Reply:
Agreed. Was thinking we'd do this later but now seems like the right time.